webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -23,39 +21,3 @@ export const notify = ({ message, type, link, sticky = isStickyType(type) }) =>
  * Clear all toast notifications
  */
 export const clear = () => store.dispatch(ToastActions.clearToasts());
-
-const railsNotificationToToastNotification = ({ link, type, message }) => {
-  const toast = { type, message };
-
-  if (link) {
-    toast.link = { href: link.href, children: link.text };
-  }
-
-  return toast;
-};
-
-const importToastNotificationsFromRails = () => {
-  const toastNotificationsContainer = $('#toast-notifications-container');
-  if (toastNotificationsContainer.length === 0) return;
-
-  const notifications = toastNotificationsContainer.data('notifications');
-  if (!notifications) return;
-
-  // notify each rails notification
-  notifications
-    .map(railsNotification =>
-      railsNotificationToToastNotification(railsNotification)
-    )
-    .forEach(toastNotification => notify(toastNotification));
-
-  // remove both jquery cache and dom entry to avoid ajax ContentLoad events
-  // reloading our toast notifications
-  toastNotificationsContainer.attr('data-notifications', '').removeData();
-};
-
-// load toast notifications from Rails on ContentLoad
-// to accommodate rails flash syntax
-$(document).on('ContentLoad', () => {
```

Comment:
I think the issue you will face with this approach that you won't catch the on content load event, and then follow up notifications will not show up. this is the original reason why it was done this way... IMHO I don't see the value of refactoring this at this stage.

Reply:
I didn't find any notifications that this might not catch, can you point me to one?
The value of refactoring this is to stop using jquery in foreman

Reply:
this might be used in plugins vs core (afair we updated all of core usage cases to call the notify methods).

regardless, the whole point of creating this interface was not to touch the actual notification react code, but keep a small jquery wrapper until we no longer create server side toast notifications.

IMHO, not the best place to spend energy in...

Reply:
Just tested this in plugins that just use `flash[:warning]` instead of the foreman wrapper, and it still works so I'm not sure where this could fail.
I think this is an simple step toward removing the jquery from foreman

Reply:
I missed your reply, let me try to explain a bit the issue:
Currently, there are 3 ways to generate toast notifications:
1. using pure JS/react/redux
2. using rails flash messages via a full page load (or initial page load)
3. using AJAX partial html response.

the refactor for this PR address only item 2, and in fact, this has been written initially to address 2 and 3.

regarding my point of refactoring, while I'm very much in favour of reducing the usage of jquery, and technical debt in general, in this particular case, I don't see the value rewriting code that was already refactored to reduce technical debt, there are a lot more! I started tracking it a long time ago at https://projects.theforeman.org/issues/23446, and would love if you guys pick items to work from there...thanks!